### PR TITLE
[T4A5][W13-B1] Song Zhiwen

### PR DESCRIPTION
**Pull request by zzzzwen**
ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.data.person;
+
+
+import seedu.addressbook.data.tag.Tag;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Tagging.java:5]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/person/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+public class Tagging {
+    private boolean sign;
+    private Person person;
+    private Tag tag;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Tagging.java:10]: where would you maintain the Tagging list?

**Comment by okkhoy** [comment]
@zzzzwen 
Some comments added. Please ack & close the PR after reading comments.

**Comment by zzzzwen** [comment]
noted

**Pull request closed without merging**